google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/Query.java

```diff
@@ -258,6 +259,23 @@ public ReadRowsRequest toProto(RequestContext requestContext) {
         .build();
   }
 
+  /**
+   * Wraps the protobuf {@link ReadRowsRequest}. This method is considered an internal implementation detail
+   * and not meant to be used by applications.
+   *
+   * <p>WARNING: ProjectId & InstanceId of {@link ReadRowsRequest} would be ignored.
```

Comment:
This is a bit alarmist maybe soften it to "Please note that the project id & instance id in the table name will be overwritten by the configuration in the BigtableDataClient"